Dockerfile

```diff
@@ -40,11 +43,25 @@ RUN apt-get update \
   libxml2-dev \
   libxmmsclient-dev \
   libxnvctrl-dev \
+  make \
   ninja-build \
   patch \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
 
+# Compile CMake, we need the latest because the bug here (for armv7 builds):
```

Comment:
Is adding this back intentional? It was removed in https://github.com/brndnmtthws/conky/commit/a533bcb8e41c57fdfa7a4aec7f3c2cd6c34734fe, so I'm wondering if this is just a merge issue.